multi-node/aws/pkg/cluster/config.go

```diff
@@ -66,8 +120,11 @@ func DecodeConfigFromFile(out *Config, loc string) error {
 
 func NewDefaultConfig(ver string) *Config {
 	return &Config{
-		ClusterName: "kubernetes",
-		ArtifactURL: DefaultArtifactURL(ver),
+		ClusterName:  "kubernetes",
+		ArtifactURL:  DefaultArtifactURL(ver),
+		VpcCIDR:      DefaultVpcCIDR,
+		SubnetCIDR:   DefaultSubnetCIDR,
+		ControllerIP: DefaultControllerIP,
```

Comment:
You don't need to set these, the defaults are handled by the cloudformation properties